tests/test_iter.py

```diff
@@ -0,0 +1,77 @@
+import json
+import os
+
+import pytest
+
+from hypothesis import (
+    given,
+    settings,
+    strategies,
+)
+
+from eth_utils import force_bytes
+
+from trie import HexaryTrie
+from trie.iter import NodeIterator
+from trie.utils.nodes import is_extension_node
+from .utils import make_random_trie
+
+
+ROOT_PROJECT_DIR = os.path.dirname(os.path.dirname(__file__))
+NEXT_PREV_FIXTURE_PATH = os.path.join(
+    ROOT_PROJECT_DIR, 'fixtures', 'TrieTests', 'trietestnextprev.json')
+RAW_NEXT_PREV_FIXTURES = [
+    (os.path.basename(NEXT_PREV_FIXTURE_PATH), json.load(open(NEXT_PREV_FIXTURE_PATH)))
+]
+NEXT_PREV_FIXTURES = [
+    ("{0}:{1}".format(fixture_filename, key), fixtures[key])
+    for fixture_filename, fixtures in RAW_NEXT_PREV_FIXTURES
+    for key in sorted(fixtures.keys())
+]
+
+
+@pytest.mark.parametrize(
+    'fixture_name,fixture', NEXT_PREV_FIXTURES,
+)
+def test_trie_next_prev_using_fixtures(fixture_name, fixture):
+    trie = HexaryTrie(db={})
+    for k in fixture['in']:
+        k = force_bytes(k)
+        trie[k] = k
+
+    iterator = NodeIterator(trie)
+    for point, _, nxt in fixture['tests']:
+        point = force_bytes(point)
+        nxt = force_bytes(nxt)
```

Comment:
You shouldn't use `force_bytes` anymore.  It's going to be deprecated due to some unfixable encoding issues.

Reply:
Should I just `.encode('utf-8')` until the new APIs land?

Reply:
Yes, I think that is fine.